### PR TITLE
[CTRL-2950] Create the ability to profile memory from within SP

### PR DESCRIPTION
**Pull request by sergey-me**
Update documentation after adding memory snapshot creation to the support tools menu.

### CODE REVIEW

**Review by quinntracy — APPROVED** [review]

File: content/en/docs/refguide/modeling/menus/help-menu/profiling.md [hunk 1]
```diff
@@ -27,3 +33,13 @@ When you are done, click **Profiling** > **Stop profiling**. Zip all the files y
 {{% alert color="info" %}}
 Profiling can be started at any time (when you start to see performance issues) and does not need to start immediately after running Studio Pro.
 {{% /alert %}}
+
+## Capture Memory Snapshot
+
+When you click **Profiling** > **Capture Memory Snapshot**, you see a dialog box that asks you to choose a location on the local disk to save the memory snapshot to. Avoid selecting a network drive.
```
Comment (MariaShaposhnikova) [inline content/en/docs/refguide/modeling/menus/help-menu/profiling.md:35]: @sergey-me , what does this option do? Could you a description of it?

File: content/en/docs/refguide/modeling/menus/help-menu/profiling.md [hunk 2]
```diff
@@ -27,3 +33,13 @@ When you are done, click **Profiling** > **Stop profiling**. Zip all the files y
 {{% alert color="info" %}}
 Profiling can be started at any time (when you start to see performance issues) and does not need to start immediately after running Studio Pro.
 {{% /alert %}}
+
+## Capture Memory Snapshot
```
Comment (MariaShaposhnikova) [inline content/en/docs/refguide/modeling/menus/help-menu/profiling.md]: Or should this be a child section of the previous one? Does Memory Proflier describe what Capture Memory Snapshot does?

File: content/en/docs/refguide/modeling/menus/help-menu/profiling.md [hunk 3]
```diff
@@ -7,12 +7,18 @@ description: "Describes how to run a performance profiler session on the local m
 ## Introduction
 
 {{% alert color="info" %}}
-This feature is not yet supported on native MacOS machines.
+These features are not yet supported on native MacOS machines.
```
Comment (MariaShaposhnikova) [inline content/en/docs/refguide/modeling/menus/help-menu/profiling.md]:
```suggestion
These features are not yet supported on native macOS machines.
```

File: content/en/docs/refguide/modeling/menus/help-menu/profiling.md (hunk 1 above)
Comment (sergey-me) [reply content/en/docs/refguide/modeling/menus/help-menu/profiling.md:35]: This option takes a snapshot of memory being consumed by SP at the moment and saves the data to a file on the local disk. The user is asked to choose the folder for the file.

Comment (sergey-me) [reply content/en/docs/refguide/modeling/menus/help-menu/profiling.md:35]: While the operation is in progress, a progress dialog is displayed. So no other interactions are required.

**Pull request merged**